internal/biz/model/common.go

```diff
@@ -112,6 +112,10 @@ func DeserializeLocalResourceId(value string) LocalResourceId {
 	return Deserialize[LocalResourceId](value)
 }
 
+func DeserializeRepresentation(value JsonObject) Representation {
+	return Representation(value)
```

Comment:
Serialization/Deserialization logic directly into domain models, which breaks the fundamental principle of keeping domain models pure and infrastructure-agnostic.